@here/harp-omv-datasource/lib/OmvDecoder.ts

```diff
@@ -80,6 +80,43 @@ export class Ring {
 
         return area / 2;
     }
+
+    getOutlineLines() {
```

Comment:
``getOutlineLines`` sounds a bit funny to me...
```suggestion
    getOutlines() {
```

Reply:
Also, cannot really say by looking at this function if it behaves as it should. Could you please add some unit tests for this?